src/drivers/luxon/driver.js

```diff
@@ -0,0 +1,149 @@
+import PropTypes from 'prop-types';
+import { DateTime } from 'luxon';
```

Comment:
this means that react-dates would cause both moment and luxon to be installed.

moment is a peer dep because react-dates, and the user, need to be using the same version of moment to ensure that things work properly. For luxon, it'd be the same requirement, which means that adding luxon would cause react-dates to have a peer dep requirement on *both* moment and luxon, and users would have to add *both* of them to their own package.json (which is a breaking change)

Reply:
yep, i get you.  it's in the same repo to test (it's all a WIP), though with this change I anticipated that every driver would be moved to its own repo... even temporal when it comes along.

Reply:
The luxon driver has been removed and will be uploaded to a separate repo.